drizzle/0050_reports.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS "reports" (
+  "id" uuid PRIMARY KEY NOT NULL,
+  "account_id" uuid NOT NULL,
+  "target_account_id" uuid NOT NULL,
+  "created" timestamp with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL,
+  "comment" text,
+  "posts" uuid [] DEFAULT '{}'::uuid [] NOT NULL
```

Comment:
ideally here we do polymorphic reporting, though the database schema for that is more complex, and I figured you'd want to keep things simple. Pixelfed is the only software I know to kind of support polymorphic reports, but they only support a single object per report.

So if I report Sally + three of her posts, then Pixelfed's design, I believe, would see this as 4 reports (Account, Post 1, Post 2, Post 3)

Probably the way to do this would be to have a `ReportObjects` table that maps object type & object ID down.

However, polymorphic reporting is only really important for receiving reports.